mileage/helpers.rb

```diff
@@ -0,0 +1,14 @@
+module CLI
+  def self.paramaterize(arg)
+    if arg.start_with? '--'
+      filter, value = arg[2..-1].split('=')
```

Comment:
`raise "Fuel, Color or Price are the only acceptable filters" unless [:price, :fuel].include?(filter)`